briskedu/brisk-learning
# Force relogin when login error is received

Summary:
If the user can't refresh their token, we should
force them through the login flow again so they can get a new refresh token.

Test Plan:
1. Login and confirm I can check AI
2. Confirm on reload I don't need to login again
3. Clear refresh_token and reload
4. Confirm I am asked to go through login flow again
